Packs/CadoResponse/Integrations/CadoResponse/CadoResponse.yml

```diff
@@ -0,0 +1,307 @@
+commonfields:
+  id: CadoResponse
+  version: -1
+configuration:
+- display: Cado Response Instance
+  additionalinfo: The URL for your Cado Response Instance
+  name: url
+  required: true
+  type: 0
+- display: API Key
+  additionalinfo: The API Key to use for connection
+  name: apikey
+  required: true
+  type: 4
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Default Cado Response Project
+  additionalinfo: Provides a fallback Project ID from Cado Response if you forget
+    to add an ID to a command. If you're not sure, don't change!
+  name: CadoResponse_DefaultProject
+  required: true
+  type: 0
+- display: Default AWS Region
+  additionalinfo: Provides a default AWS region to fallback to if you forget to add
+    it to a command.
+  name: CadoResponse_DefaultRegion
+  required: true
+  type: 0
+- display: Default S3 Bucket
+  additionalinfo: Provides a default S3 bucket to fallback to if you forget to add
+    it to a command.
+  name: CadoResponse_DefaultBucket
+  required: true
+  type: 0
+description: Automate data collection. Process data at cloud speed. Analyze with purpose.
+display: Cado Response
+name: CadoResponse
+script:
+  commands:
+  - name: cado-create-project
+    arguments:
+    - name: project_name
+      default: true
+      description: Name of the project
+      isArray: false
+      required: true
+      secret: false
+    - name: project_description
+      default: false
+      description: Description for the project
+      isArray: false
+      required: false
+      secret: false
+    deprecated: false
+    description: Create a project in Cado Response
+    execution: false
+    outputs:
+    - contextPath: CadoResponse.Project.id
+      description: The Project ID of the newly created project
+      type: Number
+  - name: cado-list-project
+    deprecated: false
+    description: Get a list of projects from Cado Response
+    execution: false
+    arguments:
+    - name: project_id
+      default: true
+      description: ID of the project to retrive
+      isArray: false
+      required: false
+      secret: false
+    - name: limit
+      default: false
+      description: Limit results to retrieve
+      isArray: false
+      required: false
+      secret: false
+    outputs:
+    - contextPath: CadoResponse.Projects.id
+      description: ID of the retrieved project
+      type: Number
+    - contextPath: CadoResponse.Projects.caseName
+      description: Name of the retrieved project
+      type: String
+    - contextPath: CadoResponse.Projects.description
+      description: Description of the retrieved project
+      type: String
+    - contextPath: CadoResponse.Projects.users
+      description: Array of users assigned to the retrieved project
+      type: Array
+    - contextPath: CadoResponse.Projects.created
+      description: Creation date of the project
```

Comment:
could you please provide an exmple of the date format you're using, for example "2020-01-01T00:11:22Z"
and all the other places we have a date value.